### PR TITLE
LPD 6890

### PR DESCRIPTION
**Pull request by ivicac**
- **LPD-6890 - Fix git error: "Host Key Verification Failed”**
- **LPD-6890 Update Faro Dockerfile**

### CODE REVIEW

**Comment by marcosapmf** [comment]
ci:test:sf

**Comment by marcosapmf** [comment]
ci:test:relevant

File: modules/dxp/apps/osb/osb-faro/Jenkinsfile [hunk 1]
```diff
@@ -18,11 +18,34 @@ pipeline {
...
-				sh "ant setup-profile-dxp"
-                sh "ant compile install-portal-snapshots"
-                sh "cd modules/dxp/apps/osb/osb-faro"
-                sh "../../../../../gradlew copyCloudResources"
+				withCredentials([string(credentialsId: 'analytics_cloud_ci_ssh', variable: 'SECRET')]) {
+					sh '''
+						export ANT_OPTS=-Xmx6g
+						export GRADLE_OPTS=-Xmx6g
+
+						echo "ssh private key: $SECRET"
```
Comment (marcellustavares) [inline modules/dxp/apps/osb/osb-faro/Jenkinsfile:26]: can we remove this echo line?

**Review by marcellustavares — CHANGES_REQUESTED** [review]

**Comment by marcellustavares** [comment]
ci:forward:force

**Review by marcellustavares — APPROVED** [review]

**Pull request closed without merging**